openfisca_core/parameters.py

```diff
@@ -120,19 +120,23 @@ def __init__(self, name, data, file_path = None):
         _validate_parameter(self, data, data_type = dict)
         self.description = None
         self.metadata = {}
+        self.documentation = None
```

Comment:
Should we also add this new attributes to parameter nodes? I'm not sure there is a good reason to do it only for leads.

Reply:
Do you see some use case for documentation on `ParameterNode`objects? 
As I see it, for now, we wouldn't use it so it would be over-engineering it.

Reply:
Yes, imagine something like: `rsa.plafonds.1_enfant`, `rsa.plafonds.2_enfants`, `rsa.plafonds.par_enfant_supp`.

In that case, it's probably more interesting to document the `rsa.plafonds` rather than the specific values.

In many case, parameters work by group and are not fully independent. In that case, the nodes may be the thing we want to document.